src/main/java/com/github/flamingsharky/darkagesexpedition/setup/Events.java

```diff
@@ -0,0 +1,17 @@
+package com.github.flamingsharky.darkagesexpedition.setup;
```

Comment:
Maybe move this files to `com.github.flamingsharky.darkagesexpedition.common.events` and rename it to `WorldEvents`